packages/amplication-client/src/Entity/RelationFkHolderField.tsx

```diff
@@ -0,0 +1,63 @@
+import { SelectField, SelectFieldProps } from "@amplication/design-system";
+import { useQuery } from "@apollo/client";
+import { useFormikContext } from "formik";
+import { useMemo } from "react";
+import * as models from "../models";
+import { GET_ENTITY_FIELD_BY_PERMANENT_ID } from "./RelatedEntityFieldField";
+
+type Props = Omit<SelectFieldProps, "options"> & {
+  entity: models.Entity;
+};
+
+const RelationFkHolderField = ({ entity, ...props }: Props) => {
+  const formik = useFormikContext<models.EntityField>();
+
+  const { data } = useQuery<{ entity: models.Entity }>(
+    GET_ENTITY_FIELD_BY_PERMANENT_ID,
+    {
+      variables: {
+        entityId: formik.values.properties.relatedEntityId,
+        fieldPermanentId: formik.values.properties.relatedFieldId,
+      },
+    }
+  );
+
+  const relatedField =
```

Comment:
add state for that and listen to changes in data

Reply:
we better use useMemo instead of state